src/backend/metal/src/command.rs

```diff
@@ -2142,8 +2158,12 @@ impl com::RawCommandBuffer<Backend> for CommandBuffer {
         };
         let mut num_colors = 0;
 
-        for (rat, clear_value) in render_pass.attachments.iter().zip(clear_values) {
-            let value = *clear_value.borrow();
+        let clear_values_iter = clear_values
+            .into_iter()
+            .map(|c| Some(*c.borrow()))
+            .chain(iter::repeat(None));
```

Comment:
I don't think the `Option` really helps here. The extra elements that we add are not any worse than the ones provided but not used, there is no need to try taking extra care of them. Instead, could just go with a default clear value or something.

Reply:
Fair enough, I was trying to avoid creating extra clear values but maybe it's not a big deal.